### PR TITLE
[MVI-834] (unit-test) check validate_json_response()

### PR DESCRIPTION
**Pull request by tulitune**
Checked validate_json_response() for the following scenarios:
1) invalid response
2) valid response:
    (a) bad http status
    (b) good http status
          (i) no header
          (ii) with header
                (I) valid json content
                (II) invalid json content
                (III) valid html content
                (IV) invalid html content

### CODE REVIEW

File: tests/support/response/test_validate.py [hunk 1]
```diff
@@ -5,31 +5,101 @@
...
 from requests.models import Response, codes
 
 response_ok = Response()
 response_ok.status_code = codes.ok
 response_bad = Response()
 response_bad.status_code = codes.bad
 
+@pytest.fixture(scope='session')
+def responses_dict():
+    print('>>>>>> responses_dict')
```
Comment (jefftune) [inline tests/support/response/test_validate.py]: @tulitune Do you want to leave the `print()` in the code

Comment (tulitune) [inline tests/support/response/test_validate.py]: @jefftune - Oops. Forgot that. It was for debugging.

**Comment by jefftune** [comment]
@tulitune LGTM provided you have removed the `print()`

**Comment by tulitune** [comment]
@jefftune, you have my word.

**Pull request merged**